nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
```

Comment:
Yes. I usually don't complain if that is how the file is already but at some point we should clean them out.